.golangci.yaml

```diff
@@ -87,9 +87,16 @@ linters:
 formatters:
   enable:
     - "gci"
+    - "gofmt"
```

Comment:
Does adding gofmt here mean there's yet another tool that's processing the entire codebase?

Not a reason not to do it in my head but wanna be aware of the tradeoffs.

Reply:
Probably? I'm fairly certain goreleaser runs everything in parallel.